books/free-programming-books-se.md

```diff
@@ -15,11 +15,6 @@
 * [Programmera spel i C++ för nybörjare](https://sv.wikibooks.org/wiki/Programmera_spel_i_C%2B%2B_f%C3%B6r_nyb%C3%B6rjare) - Wikibooks
 
 
-### MATLAB
-
-* [Introduktion till MATLAB](https://www.liber.se/plus/E470523401.pdf) (PDF)
-
-
```

Comment:
An alternative hosted at LINKÖPINGS UNIVERSITET: https://www.cvl.isy.liu.se/education/undergraduate/TSKS08/matlab-1/resurser/


Apply this patch to restore resource
```suggestion
### MATLAB

* [Introduktion till MATLAB (2004)](https://www.cvl.isy.liu.se/education/undergraduate/TSKS08/matlab-1/Matlabintro_sve.pdf) - Liber AB, Lennart Harnefors, Johnny Holmberg, Joop Lundqvist (PDF)


```

Reply:
Suggestion applied at #6595